src/cast_core/src/CollectionScanner.cpp

```diff
@@ -417,6 +417,8 @@ void CollectionScanner::run() {
     std::optional<bsoncxx::types::b_timestamp> readClusterTime;
     for (auto&& config : _loop) {
         for (const auto&& _ : config) {
+            BOOST_LOG_TRIVIAL(debug)
+                << "Starting " << this->defaultName() << "::" << this->name() << "::" << this->id() << " execution";
```

Comment:
Tried to make this an `actorInfo` variable at actor level but was getting linker errors for `defaultName` (after uncommenting [this](https://github.com/mongodb/genny/blob/30598a24dfd223a8a18e4d51def6c20ca4916995/src/gennylib/include/gennylib/Actor.hpp#L80). Maybe there's a way around it?

Otherwise I guess this saves some space to not have an extra variable that's only used for debug logging?